src/overlays/actors/ovl_En_Ru2/z_en_ru2.c

```diff
@@ -16,32 +16,61 @@ void EnRu2_Destroy(Actor* thisx, PlayState* play);
 void EnRu2_Update(Actor* thisx, PlayState* play);
 void EnRu2_Draw(Actor* thisx, PlayState* play);
 
-void func_80AF2CB4(EnRu2* this, PlayState* play);
-void func_80AF2CD4(EnRu2* this, PlayState* play);
-void func_80AF2CF4(EnRu2* this, PlayState* play);
-void func_80AF2D2C(EnRu2* this, PlayState* play);
-void func_80AF2D6C(EnRu2* this, PlayState* play);
-void func_80AF2DAC(EnRu2* this, PlayState* play);
-void func_80AF2DEC(EnRu2* this, PlayState* play);
-void func_80AF3144(EnRu2* this, PlayState* play);
-void func_80AF3174(EnRu2* this, PlayState* play);
-void func_80AF31C8(EnRu2* this, PlayState* play);
-void func_80AF3604(EnRu2* this, PlayState* play);
-void func_80AF3624(EnRu2* this, PlayState* play);
-void func_80AF366C(EnRu2* this, PlayState* play);
-void func_80AF36AC(EnRu2* this, PlayState* play);
-void func_80AF3BC8(EnRu2* this, PlayState* play);
-void func_80AF3C04(EnRu2* this, PlayState* play);
-void func_80AF3C64(EnRu2* this, PlayState* play);
-void func_80AF3CB8(EnRu2* this, PlayState* play);
-void func_80AF3D0C(EnRu2* this, PlayState* play);
-void func_80AF3D60(EnRu2* this, PlayState* play);
-
-void func_80AF3F14(EnRu2* this, PlayState* play);
-void func_80AF3F20(EnRu2* this, PlayState* play);
-void func_80AF321C(EnRu2* this, PlayState* play);
-
-void func_80AF2AB4(EnRu2* this, PlayState* play);
+void EnRu2_Action00(EnRu2* this, PlayState* play);
+void EnRu2_Action01(EnRu2* this, PlayState* play);
+void EnRu2_Action02(EnRu2* this, PlayState* play);
+void EnRu2_Action03(EnRu2* this, PlayState* play);
+void EnRu2_Action04(EnRu2* this, PlayState* play);
+void EnRu2_Action05(EnRu2* this, PlayState* play);
+void EnRu2_Action06(EnRu2* this, PlayState* play);
+void EnRu2_Action07(EnRu2* this, PlayState* play);
+void EnRu2_Action08(EnRu2* this, PlayState* play);
+void EnRu2_Action09(EnRu2* this, PlayState* play);
+void EnRu2_Action10(EnRu2* this, PlayState* play);
+void EnRu2_Action11(EnRu2* this, PlayState* play);
+void EnRu2_Action12(EnRu2* this, PlayState* play);
+void EnRu2_Action13(EnRu2* this, PlayState* play);
+void EnRu2_Action14(EnRu2* this, PlayState* play);
+void EnRu2_Action15(EnRu2* this, PlayState* play);
+void EnRu2_Action16(EnRu2* this, PlayState* play);
+void EnRu2_Action17(EnRu2* this, PlayState* play);
+void EnRu2_Action18(EnRu2* this, PlayState* play);
+void EnRu2_Action19(EnRu2* this, PlayState* play);
```

Comment:
do these actions directly correspond to the enum values below? If so maybe it would be better to name these with e.g. `EnRu2_AwaitBlueWarp`. Some of the enum names aren't very verb-y though so I'm not sure how this will shake out. I'd be fine with the way it is now too, it's already an improvement